ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-ref.yaml

```diff
@@ -0,0 +1,34 @@
+ref:
+  as: cucushift-installer-check-vsphere-customized-resource
+  from_image:
+   namespace: ocp
+   name: "4.14"
+   tag: upi-installer
+  commands: cucushift-installer-check-vsphere-customized-resource-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: CONTROL_PLANE_REPLICAS
+    default: "3"
+    documentation: |-
+      the number of the master nodes for ipi on vsphere cluster.
+  - name: COMPUTE_NODE_REPLICAS
+    default: "3"
+    documentation: |-
+      the number of the worker nodes for ipi on vsphere cluster.
+  - name: CONTROL_PLANE_DISK_SIZE
```

Comment:
Suggest to keep consistent with the default values in `ipi-conf-vsphere-customize-resource`